models/stock_picking.py

```diff
@@ -67,6 +67,8 @@ def update_picking(self, expected_package_name=None, **kwargs):
             TODO: finish implement and add parameter at README
         """
         # extra_context = {}
+
+        kwargs.update({'real_time_update': self.picking_type_id.u_validate_real_time})
```

Comment:
Why not using `validate_real_time`, to follow the model's attribute?

Reply:
Ah, I see, the attribute is then used by the `super` method, in udes_core and you don't want to change that (but there's an open PR...).